### ISSUE TITLE
How to update ota with out reinstall from 0?

### ISSUE THREAD

**Issue by toshibochan** [HomeACcessoryKid/LCMdev#5]
Is possible to update ota bin without erase the device?

**Comment by HomeACcessoryKid** [comment]
yes, it is one of the things I was about to document on https://github.com/HomeACcessoryKid/life-cycle-manager. Check it out after today
